### PR TITLE
feat(rn): Add React Native Metro Bundler page

### PR DESCRIPTION
**Pull request by krystofwoldrich**
Adds docs about the Sentry Metro Serializer released in https://github.com/getsentry/sentry-react-native/releases/tag/5.11.0 and https://github.com/getsentry/sentry-react-native/releases/tag/5.16.0-alpha.1

### CODE REVIEW

**Review by lizokm — COMMENTED** [review]
Hey there! Thanks for writing this. Shana and I are both not feeling well today, but will look at this as soon as we're better. Thanks for being patient with us!

**Comment by krystofwoldrich** [comment]
@lizokm Thank you for the message. Get well soon.

File: src/platforms/react-native/manual-setup/metro.mdx [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+title: Metro
+sidebar_order: 3
+redirect_from:
+  - /platforms/react-native/advanced-setup/
+description: "Learn about the Metro bundler and how to configure it for your your application with Sentry React Native SDK."
+---
+
+Sentry React Native SDK package ships with a Sentry Metro Serializer which allows you to automatically generated Debug IDs for your applications bundles. This is crucial for source maps to work properly with Sentry.
+This page will guide you through the process of setting up the Metro Plugin for your application.
```
Comment (lizokm) [inline src/platforms/react-native/manual-setup/metro.mdx:10]:
```suggestion
Sentry's React Native SDK package ships with a Sentry Metro Serializer which allows you to automatically generate Debug IDs for your applications' bundles. This is crucial for making source maps work properly with Sentry.
This page will guide you through the process of setting up the Metro Plugin for your application.
```

File: src/platforms/react-native/manual-setup/metro.mdx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+## Prerequisities
+
+- [Sign up for an account](https://sentry.io/signup/)
+- [Install Sentry React Native SDK](/platforms/react-native) version 5.11.0 or newer
+  - Expo is supported from the SDK version 5.16.0-alpha.1
```
Comment (lizokm) [inline src/platforms/react-native/manual-setup/metro.mdx:16]:
```suggestion
  - Expo is supported from SDK version 5.16.0-alpha.1
```

File: src/platforms/react-native/manual-setup/metro.mdx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+## Configuration
+
+The Sentry React Native SDK allows multiple ways to configure the Sentry Metro Serializer depending on your current use of `customeSerializer` in your Metro configuration.
```
Comment (lizokm) [inline src/platforms/react-native/manual-setup/metro.mdx:20]:
```suggestion
The Sentry React Native SDK allows multiple ways to configure the Sentry Metro Serializer, depending on your current use of `customeSerializer` in your Metro configuration.
```

File: src/platforms/react-native/manual-setup/metro.mdx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  serializer: {
+    customSerializer: createSentryMetroSerializer(),
+  },
+};
+
+const finalConfig = mergeConfig(config, sentryConfig);
+module.exports = withExpoSerializers(finalConfig);
+```
+
+### Wrap your custom serializer
```
Comment (lizokm) [inline src/platforms/react-native/manual-setup/metro.mdx:61]:
```suggestion
### Wrap Your Custom Serializer
```

File: src/platforms/react-native/manual-setup/metro.mdx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+```typescript
+export type Bundle = {
+  modules: Array<[id: number, code: string]>;
+  post: string;
+  pre: string;
+};
+```
+
+## Throubleshooting
```
Comment (lizokm) [inline src/platforms/react-native/manual-setup/metro.mdx:101]:
```suggestion
## Troubleshooting
```

**Review by lizokm — COMMENTED** [review]
Love the prerequisites, troubleshooting and giving users more of an information ramp!!
Made a few language tweaks, but otherwise this looks great!

**Pull request merged**